src/components/PopupWithForm.js

```diff
@@ -4,35 +4,44 @@ class PopupWithForm extends Popup {
   constructor(popupSelector, handleFormSubmit) {
     super({ popupSelector });
     this._handleFormSubmit = handleFormSubmit;
-    this._form = document.querySelector(popupSelector).querySelector("form");
+    this._form = this._popupElement.querySelector("form");
     this._inputList = [...this._popupElement.querySelectorAll("input")];
+    this.submitBtn = this._popupElement.querySelector(".modal__save-button");
+
   }
 
   setPreviewedValues(values) {
     this._inputList.forEach((input) => {
       input.value = values[input.name];
+
     });
   }
   _getInputValues() {
     const inputValues = {};
 
-    this._inputList.forEach((input) => {
+    this._inputList.forEach((input) => { // look at this 
       inputValues[input.name] = input.value;
     });
-
     return inputValues;
   }
 
+  open(){
+    super.open();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.